pandas/tests/tseries/test_libfrequencies.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+
+import pandas.util.testing as tm
+
+from pandas.tseries import offsets
+from pandas._libs.tslibs.frequencies import (get_rule_month,
+                                             _period_str_to_code,
+                                             _INVALID_FREQ_ERROR,
+                                             is_superperiod, is_subperiod)
+
+
+def assert_aliases_deprecated(freq, expected, aliases):
```

Comment:
do we have these listed in deprecations for removal? @gfyoung ?

we should blow these away in 0.23.0, can you create an issue an add to the list?